.infrastructure/todoapp-pod.yml

```diff
@@ -21,3 +30,37 @@ spec:
         port: 8080
       initialDelaySeconds: 5
       periodSeconds: 5
+
+---
```

Comment:
Consider using a Deployment or ReplicaSet instead of defining multiple pods manually. This approach provides better scalability and management of pod replicas.